_posts/2021-04-08-alekos-bdk.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Alekos Filini on Bitcoin Development Kit"
+permalink: /blog/2021/04/08/bitcoin-development-kit/
+layout: post
+author: alekos
+name: "Alekos"
+alt: Alekos Filini on Bitcoin Development Kit
+category: "News"
+description: Alekos Filini on Bitcoin Development Kit.
+---
+
+<img src="/assets/images/2021-04-08-bitcoin-development-kit.svg" />
+
+While a very high level understanding of Bitcoin can be enough to grasp how
+value is moved across the network, taking this knowledge and using it to build a
+real, user-ready product can prove to be very hard, due to the high security
+standards that need to be met, and also in part because of all the little quirks
+the Bitcoin protocol has.
+
+Over the years many different libraries have been released to help. Application
+developers can work more effectively on their products. Those libraries come in a
```

Comment:
Suggestion: Application developers can work more effectively on their products and leave the technical and security concerns to the underlying library.